ra-tls/README.md

```diff
@@ -1,16 +1,31 @@
 # Configure SGX RA settings
+## For EPID RA
 ``` shell
 export SPID=<hex string>
 export EPID_SUBSCRIPTION_KEY=<hex string>
 export QUOTE_TYPE=<SGX_LINKABLE_SIGNATURE | SGX_UNLINKABLE_SIGNATURE>
 ```
+## For DCAP RA
+```shell
+mkdir -p $PATH_TO_DCAP_SOURCE
+cd $PATH_TO_DCAP_SOURCE
+git clone https://github.com/intel/SGXDataCenterAttestationPrimitives/
+export SGX_DCAP=$PATH_TO_DCAP_SOURCE/SGXDataCenterAttestationPrimitives
+```
 
 # Build
+## For EPID RA
 ``` shell
 sudo yum install -y glibc-static
 cd $src/ra-tls
 make
 ```
+## For DCAP RA
```

Comment:
Add an empty line between title and content. Plz apply this principle to the following changes.